python/src/mas/cli/update/app.py

```diff
@@ -437,21 +446,16 @@ def detectUDS(self) -> None:
                         self.showUDSUpdateNotice()
                         self.fatalError(f"By choosing {self.getParam('mas_catalog_version')} you must provide the storage class to use for the migration to DRO using '--dro-storage-class' when using '--no-confirm'")
                     else:
-                        h.stop_and_persist(symbol=self.successIcon, text="IBM User Data Services needs to be migrated to IBM Data Reporter Operator")
-                        self.showUDSUpdateNotice()
```

Comment:
Removing the notice here means we could just exit without much feedback to the user as to why

Reply:
Added the notice here to show in the case of UDS installed